airbyte-commons-temporal/src/main/java/io/airbyte/commons/temporal/TemporalClient.java

```diff
@@ -400,7 +400,9 @@ public TemporalResponse<StandardSyncOutput> submitSync(final long jobId, final i
         .withState(config.getState())
```

Comment:
I noticed source_id and destination_id wasn't passed in here. But is passed in other places where we build this struct. @pedroslopez was that intentional?

Reply:
Not intentional. I didn't know submitSync was a thing when working on the original PR that introduced them. 

I've actually been recently trying to understand where this submitSync method comes into play... when does this get executed? From what I've seen we're always using the inputs as generated by the GenerateInputActivity in actual sync jobs.

Reply:
tbh, i've lost track of it as well. @benmoriceau is this something you understand well?